adapters/tappx/tappx_test.go

```diff
@@ -38,13 +39,16 @@ func TestTsValue(t *testing.T) {
 
 	bidderTappx := bidder.(*TappxAdapter)
 
-	var test int
-	test = 0
+	test := 0
 	var tappxExt openrtb_ext.ExtImpTappx
 	tappxExt.Endpoint = "DUMMYENDPOINT"
 	tappxExt.TappxKey = "dummy-tappx-key"
 
 	url, err := bidderTappx.buildEndpointURL(&tappxExt, test)
+	if err != nil {
+		require.NoError(t, err, "buildEndpointURL")
```

Comment:
No need for the `return` statement because `require.NoError` stops the execution of `TestTsValue` if it fails. Can we remove?
```
47     url, err := bidderTappx.buildEndpointURL(&tappxExt, test)
48 -   if err != nil {
49         require.NoError(t, err, "buildEndpointURL")
50 -       return
51 -   }
adapters/tappx/tappx_test.go [RO]
```

Reply:
changed in 6d4f03aa

Reply:
> Why don't we use require.NoError in the next lines, after match, err := regexp.MatchString? Should I change it as well (can do in a follow-up PR)?

I completely agree. Sure, it's better to do in a follow up PR. Thanks for the change.